src/components/AllCards/AllCards.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Card from "./Card/Card";
+
+export default class AllCards extends Component {
+  render() {
+    const { cards, handleClick, filterText } = this.props;
+    let list = cards;
+    if (filterText) {
+      list = cards.filter((card) => {
+          return card.name.toLowerCase().includes(filterText);
+      });
+    }
+
+    return (
+      <div>
+        {list.map(card => {
+          return (
+            <Card
+              key={card.id}
+              card={card}
+              handleClick={id => handleClick(id)}
```

Comment:
In such cases we can pass the function directly `handleClick={handleClick}` no neeed to wrap it into fat arrow. It also has minor performance cost, as inline fat arrows are recreated on every render.